include/miopen/miopen.h

```diff
@@ -4950,6 +4950,129 @@ MIOPEN_EXPORT miopenStatus_t miopenCTCLoss(miopenHandle_t handle,
                                            void* workSpace,
                                            size_t workSpaceSize);
 
+/*! @brief Helper function to query the minimum workspace size required by the hinge embedding loss
+ * call
+ *
+ * @param handle                   MIOpen Handle (input)
+ * @param iDesc                    Tensor descriptor for input tensor (input)
+ * @param tDesc                    Tensor descriptor for target tensor (input)
+ * @param oDesc                    Tensor descriptor for output tensor (input)
+ * @param sizeInBytes              Pointer to data to return the minimum workspace size
+ * @return                         miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t
+miopenGetHingeEmbeddingLossForwardWorkspaceSize(miopenHandle_t handle,
```

Comment:
If you use variables too abbreviatedly, it's hard to know what they are. in and input are fine, but i is too short. Likewise, o is too short compared to output and out, so it is difficult to understand at once.
However, t is a variable name that is not often used, so it is okay.

Reply:
I have updated variable name: i => input, t => target, o => output, dO => doutput, dI => dinput